scipy/stats/tests/test_stats.py

```diff
@@ -2253,6 +2253,96 @@ def test_ttest_ind():
         np.seterr(**olderr)
 
 
+def test_ttest_ind_permutations():
```

Comment:
The repetition within this test indicates to use a pytest parametrize annotation. 

Suggest to do the tests with those (there are lots of examples in tests in scipy.stats).

Reply:
Done.